proto/plan.proto

```diff
@@ -83,6 +83,12 @@ message ColRef {
 	string	name	= 3;
 }
 
+// Reference a column by its internal table id and column id
```

Comment:
what is the difference between between colref and rawcolref?     so it has to be a real table?   how about view?

Reply:
ColRef is used in plan. Its fields are relative positions.

RawColRef is used in TableDef itself, i.e. partition info, (unique) index info. Its fields are internal table_id and column_id which are generated by cn.

Their fields are all integers. But I think it's better to use distinct structs, to avoid unperceived misuse.